bundle/manifests/prometheus-exporter-operator.clusterserviceversion.yaml

```diff
@@ -309,8 +309,8 @@ spec:
   - name: GitHub
     url: https://github.com/3scale-sre/prometheus-exporter-operator
   maintainers:
-  - email: 3scale-operations@redhat.com
-    name: 3scale Ops
+  - email: 3scale-sre+olm@redhat.com
```

Comment:
Does this mail address even exist?

Reply:
We are using 3scale-sre+SERVICE@redhat.com for almost everything.